kotlin/workflow-core/src/main/java/com/squareup/workflow/StatefulWorkflow.kt

```diff
@@ -75,7 +75,8 @@ abstract class StatefulWorkflow<
    * initial state.
    *
    * @param snapshot
-   * If the workflow is being created fresh, [snapshot] will be null.
+   * If the workflow is being created fresh, OR the workflow is being restored from an empty
+   * [Snapshot] (i.e. [snapshotState] returned [Snapshot.EMPTY]), [snapshot] will be null.
```

Comment:
i.e. [Snapshot.bytes] returned an empty [ByteString], probably because [snapshotState] returned [Snapshot.EMPTY]

Reply:
Pro tip: use the "Insert a suggestion" button and then I can apply your suggestion with a button click 😉 
![image](https://user-images.githubusercontent.com/101754/65282139-09e8d600-dae9-11e9-8826-6c27d91c6751.png)

Reply:
Ugh, way too hard to figure out where the line break boundary is, that's a useless feature here.